docs/source/ModuleSystem/EasyBuildModules.md

```diff
@@ -0,0 +1,71 @@
+# EasyBuild Modules
+
+These are the modules that are not-quite-the-default yet. EasyBuild enforces a nice, strict versioning. Compilers/toolchains are programs and have their own bugs. **THESE MODULES ARE NOT USABLE BY DEFAULT**.
```

Comment:
Hmm, let's get it through this time. 
A note to @robqiao improves this section next update.